test/distributed/_composable/fsdp/test_fully_shard_state_dict.py

```diff
@@ -105,7 +105,7 @@ def test_dp_state_dict_cpu_offload(self):
 
         # lazy init without error
         inp = torch.rand((mlp_dim, mlp_dim), device="cuda")
-        model(inp)
+        model(inp).sum().backward()
```

Comment:
the error only shows up in backward when we add cpu gradient to cuda DTensor param